compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/messages/MessageCollector.kt

```diff
@@ -21,9 +21,21 @@ interface MessageCollector {
 
     fun report(severity: CompilerMessageSeverity, message: String, location: CompilerMessageLocation? = null)
 
+    fun reportExt(severity: CompilerMessageSeverity, message: String, location: CompilerMessageExtendedLocation? = null) {
```

Comment:
I do not like `...Ext` naming - remind me of bad MS APIs habits. I'd rather use `@JvmName` to resolve java ambiguity`.